src/components/subscriptions/edit/EditSubAddons.js

```diff
@@ -0,0 +1,206 @@
+                            onAddAddonsSelected={onAddAddonsSelected}
+                            {...arrayHelpers}
+                        />
+                    )}
+                />
+            )}
+        </>
+    );
+}
+
+function Addons(props) {
+    const {
+        form: { values },
+        isFetchingSubAddons,
+        name,
+        parentId,
+    } = props;
+    const { t } = useTranslation("subscriptions");
+    const addons = getIn(values, name);
+    const tableId = buildID(parentId, ids.SUB_ADDONS.EDIT_ADDONS_TABLE);
+    return (
+        <>
+            <Table>
+                {(!addons || addons.length === 0) && (
+                    <EmptyTable
+                        message={t("noAddons")}
+                        numColumns={TABLE_COLUMNS.length}
+                    />
+                )}
+                <DETableHead
+                    baseId={parentId}
+                    columnData={TABLE_COLUMNS}
+                    rowsInPage={addons ? addons.length : 0}
+                    selectable={false}
+                />
+                {isFetchingSubAddons ? (
+                    <LoadingMask columns={TABLE_COLUMNS} tableId={tableId} />
+                ) : (
+                    <TableBody>
+                        {addons &&
+                            addons.length > 0 &&
+                            addons.map((addon, index) => {
+                                let resourceInBytes =
+                                    addon.resource_type.toLowerCase() ===
+                                    "bytes";
+                                return (
+                                    <DERow tabIndex={-1} key={index}>
+                                        <TableCell>
+                                            <Typography>
+                                                {addon.name}
+                                            </Typography>
+                                        </TableCell>
+                                        <TableCell>
+                                            <Typography>
+                                                {resourceInBytes
+                                                    ? "GiB"
+                                                    : addon.resource_type}
+                                            </Typography>
+                                        </TableCell>
+                                        <TableCell>
+                                            <Field
+                                                name={`${name}.${index}.amount`}
+                                                component={FormNumberField}
+                                                disabled
+                                                fullWidth={false}
+                                                id={ids.SUB_ADDONS.NAME_FIELD}
+                                                required
+                                            />
```

Comment:
I don't think this needs to be changed for this PR, and I'm not sure if there are plans in the future to allow admins to edit these values, but if not, then I think it makes these values a little harder to read displaying them in disabled form fields.

Formik would not be needed at all to display these values if we do not plan to allow admins to edit these values, and they can be displayed in a simpler listing, similar to `src/components/subscriptions/edit/Usages.js`.

Reply:
One of the requirements for this UI is that the admins can edit these values. That will be included in the next PR. However, I have noted that Formik fields wouldn't be needed if the display could be handled with a listing :).

Reply:
I suspected that might be the case 👍 but I thought I'd mention it anyway, just in case 😅